Source/carthage/Extensions.swift

```diff
@@ -55,21 +56,50 @@ extension GitURL: ArgumentType {
 	}
 }
 
+extension Color.Wrap {
+	func autowrap(string: String) -> String {
+		return Formatting.color ? self.wrap(string) : string
+	}
+}
+
+internal struct Formatting {
+	static let color = Terminal.isTTY && !Terminal.isDumb
+	
+	static let bulletin = Color.Wrap(foreground: .Blue, style: .Bold)
+	static let bullets: String = {
+		return bulletin.autowrap("***") + " "
+	}()
+	
+	static let URL = [StyleParameter.Underlined] as Color.Wrap
+	static let projectName = [StyleParameter.Bold] as Color.Wrap
+	static let path = Color.Wrap(foreground: .Yellow)
+	
+	static func quote(string: String) -> String {
+		return Color.Wrap(foreground: .Green).autowrap("\u{0022}" + string + "\u{0022}")
```

Comment:
That actually just interpolates to `"`. 

[⌘F for “arbitrary Unicode scalar” in these Swift docs](https://developer.apple.com/library/prerelease/ios/documentation/Swift/Conceptual/Swift_Programming_Language/StringsAndCharacters.html).